legacy/toml-config/src/main/java/io/zeebe/legacy/tomlconfig/LegacyConfigurationSupport.java.orig

```diff
@@ -0,0 +1,233 @@
+/*
```

Comment:
I guess this wasn't meant to be here :sweat_smile:

Reply:
???

Reply:
Do we want to commit a .java.orig file?

Reply:
Oh, that... done